comeet/UI/MainMenu/MainMenuViewController.swift

```diff
@@ -30,10 +36,60 @@ class MainMenuViewController: BaseViewController {
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         guard let identifier = segue.identifier,
             let viewModel = viewModel else {
+                print("")
                 return
         }
         Router.prepare(identifier: identifier, destination: segue.destination, sourceViewModel: viewModel)
     }
+    
+    
+    
+    @IBAction func openAgenda(_ sender: Any) {
+        
+        let viewController = SSCalendarDailyViewController(events: generateEvents())!
```

Comment:
I think we should have our view controller that inherits from SSCalendarDailyViewController. Something like MyAgendaMonthViewController.swift. And use a view model to do the transformation between the meeting model and SSDayNode. Also this will help us use a simple segue on the story board and handle the dependency injection in  Router.prepare(...) as in the other VCs/VMs.